shapeshift/mobile-app
# feat: whitelist chatwoot

this whitelists the chatwoot url to allow the webview to open it within the app in an iframe

this prevents it from opening in a new tab when the feature flag is enabled in web and allow us to ship this feature for support

to test

1. open simulator/log in with demo wallet
2. click hamburger menu, settings
3. click settings heading 5 times
4. scroll down to debugging section and choose `yeet` as environment (yeet is web `develop` branch with chatwoot feature flag enabled) 
![image](https://github.com/shapeshift/mobile-app/assets/88504456/889f73d5-72f3-4748-a250-ae30b8da8fea)
6. click hamburger menu and click feedback & support, then get support
7. see that chatwoot opens within the app, and doesn't open a new browser tab in safari

![image](https://github.com/shapeshift/mobile-app/assets/88504456/87ff8cfd-5ad6-4a85-84ec-aff6d52db439)
